packages/cozy-client/package.json

```diff
@@ -41,15 +41,18 @@
   "devDependencies": {
     "@babel/cli": "7.12.8",
     "@cozy/codemods": "^1.1.0",
+    "@material-ui/core": "4",
     "@testing-library/react": "^10.0.1",
     "@testing-library/react-hooks": "^3.2.1",
     "babel-plugin-search-and-replace": "1.1.0",
     "btoa": "1.2.1",
     "cozy-logger": "1.6.0",
-    "cozy-ui": "^34.0.0",
+    "cozy-ui": "48.0.0",
+    "date-fns": "2.19.0",
     "jsdoc-plugin-intersection": "^1.0.4",
     "mockdate": "^2.0.2",
     "react": "16.13.1",
+    "react-inspector": "5.1.0",
```

Comment:
Since this is not a "cozy" dep, should we add a note about that in  ./devtools.md ?

Reply:
I've added a note on necessary dependencies, thanks ! 

See 97383e785fb53d08634b631555eb05b292df5230